<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.web.method.support.InvocableHandlerMethodTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2020-12-11T19:56:43" hostname="DESKTOP-TMNPDT1" time="0.027">
  <properties/>
  <testcase name="resolveArg()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.003"/>
  <testcase name="cannotResolveArg()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.002"/>
  <testcase name="resolveProvidedArg()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.001"/>
  <testcase name="illegalArgumentException()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.003"/>
  <testcase name="invocationErrorMessage()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.002"/>
  <testcase name="invocationTargetException()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.005"/>
  <testcase name="resolveProvidedArgFirst()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.001"/>
  <testcase name="exceptionInResolvingArg()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.001"/>
  <testcase name="resolveNoArgValue()" classname="org.springframework.web.method.support.InvocableHandlerMethodTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
